src/ui_basic/color_chooser.cc

```diff
@@ -157,6 +225,11 @@ ColorChooser::ColorChooser(Panel* parent,
      hbox_(&main_box_, panel_style_, 0, 0, Box::Horizontal),
      buttonsbox_(&main_box_, panel_style_, 0, 0, Box::Horizontal),
      vbox_(&hbox_, panel_style_, 0, 0, Box::Vertical),
+     box_r_(&vbox_, panel_style_, 0, 0, Box::Horizontal),
+     box_g_(&vbox_, panel_style_, 0, 0, Box::Horizontal),
+     box_b_(&vbox_, panel_style_, 0, 0, Box::Horizontal),
+     palette1_(&vbox_, panel_style_, 0, 0, Box::Horizontal),
+     palette2_(&vbox_, panel_style_, 0, 0, Box::Horizontal),
```

Comment:
please rename and have a consistent style. all boxes have "box" in the name, these two do not. also palette1, pallette2 and pallette are not very descriptive.